src/js/utils/supportedActivitiesUtils.js

```diff
@@ -0,0 +1,8 @@
+import ActivityCode from 'consts/activityCode';
+
+const isRequestFromWard = (currentLocationId, destinationId, supportedActivities) =>
```

Comment:
Technically now it can be not from `Ward` too

Reply:
@kchelstowski at least add a comment explaining that `Ward` in this context does not mean the location type but specific set of supported activities

Reply:
@kchelstowski nvm, I pushed the comment